src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -470,6 +492,42 @@ public String getQuestionResultStatisticsCsv(
                + ", Average Points" + Const.EOL
                + fragments + Const.EOL;
     }
+    
+    /**
+     * Puts recipients from an unsorted map to a sorted map
+     * 
+     * @param recipientMapping      Original map containing recipients
+     * @param identifierMap         Helper map to retrieve email from name concatenated with email string
+     * @param sortedOptionPoints    Sorted map to contain recipient info, recipient concatenated with email used as key
+     * @param bundle
+     */
+    private void putRecipientsInSortedMap(Map<String, List<Integer>> recipientMapping, Map<String, String> identifierMap,
+            Map<String, List<Integer>> sortedOptionPoints, FeedbackSessionResultsBundle bundle) {
```

Comment:
Now that the method name has increased in length, the following would be neater:
```
private void putRecipientsInSortedMap(
        Map<String, List<Integer>> recipientMapping, Map<String, String> identifierMap,
        Map<String, List<Integer>> sortedOptionPoints, FeedbackSessionResultsBundle bundle) {
```